tests/src/lib.rs

```diff
@@ -15,9 +15,5 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-pub mod arrow;
-pub mod db;
-pub mod tables;
-pub mod utils;
-
-pub use tempfile;
+#[allow(dead_code)]
+pub fn this_file_intentionally_left_blank() {}
```

Comment:
Why do we need thsi file? Just curious

Reply:
a rust crate is either a binary or a library.  I chose a library for this crate, and I figured having a `lib.rs` with something in it was better than a zero-byte `lib.rs`.